### PR TITLE
Import and adapt the snapcraft.yaml updating automation

### PR DESCRIPTION
**Pull request by aieri**
This workflow should allow us to generate PRs to update snapcraft.yaml
in all release branches of snap-tempest

### CODE REVIEW

File: .github/workflows/update-snapcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          sudo apt-get install --yes tox
+      - name: Update Snapcraft YAML
+        id: update_snapcraft
+        run: |
+          tox -c ${{ env.AUTOMATION_REPO_PATH }}/tox.ini -e release -- \
+              --input ${{ env.SNAP_REPO_PATH }}/snap/snapcraft.yaml \
+              --output ${{ env.SNAP_REPO_PATH }}/snap/snapcraft.yaml \
+              --release ${{ inputs.openstack-release }} \
+              --manual-requirements ${{ env.SNAP_REPO_PATH }}/requirements-manual.txt \
+              --excluded-plugins ${{ env.SNAP_REPO_PATH }}/excluded-plugins.txt
```
Comment (chanchiwai-ray) [inline .github/workflows/update-snapcraft.yaml:40]: These envs should now be `env.AUTOMATION_REPO_PATH`?

**Review by chanchiwai-ray — CHANGES_REQUESTED** [review]
Spotted a typo (maybe?)

File: .github/workflows/update-snapcraft.yaml (hunk 1 above)
Comment (aieri) [reply .github/workflows/update-snapcraft.yaml:40]: no, it's meant to be that way, to allow us to set per-OpenStack-release manual requirements and excluded plugins. If those paths don't exist, the default kicks in and the ones from AUTOMATION_REPO_PATH are taken

Comment (chanchiwai-ray) [reply .github/workflows/update-snapcraft.yaml:40]: I see thanks for the clarification!

**Review by chanchiwai-ray — DISMISSED** [review]
LGTM (at least by eyes)

**Review by chanchiwai-ray — APPROVED** [review]

**Pull request merged**